frontend/www/js/omegaup/components/contest/AddProblem.vue

```diff
@@ -241,12 +289,19 @@ export default class AddProblem extends Vue {
     this.$emit('runs-diff', this.alias, versions, selectedCommit);
   }
 
-  get addProblemButtonLabel(): string {
+  get problemToUpdate(): boolean {
     for (const problem of this.problems) {
       if (this.alias === problem.alias) {
-        return T.wordsUpdateProblem;
+        return true;
       }
     }
+    return false;
+  }
```

Comment:
```suggestion
  get isUpdate(): boolean {
    return this.problems.some((problem) => problem.alias === this.alias);
  }
```

`problemToUpdate` no suena booleanoso (pensaría que te regresa el problema que se va a editar)

Reply:
Es cierto. Se cambia.